spring-restdocs-restassured/src/test/java/org/springframework/restdocs/restassured/RestAssuredRestDocumentationIntegrationTests.java

```diff
@@ -113,10 +115,29 @@ public void curlSnippetWithContent() throws Exception {
 				new File(
 						"build/generated-snippets/curl-snippet-with-content/curl-request.adoc"),
 				is(snippet(asciidoctor()).withContents(codeBlock(asciidoctor(), "bash")
-						.content("$ curl 'http://localhost:" + this.port + "/' -i "
+						.content(String.format("$ curl 'http://localhost:" + this.port + "/' -i "
+								+ "-X POST \\%n -H 'Accept: application/json' "
+								+ "\\%n -H 'Content-Type: " + contentType + "' "
+								+ "\\%n -d 'content'")))));
+	}
+
+	@Test
+	public void curlSnippetWithContentSpaceSeparator() throws Exception {
```

Comment:
It's good to have tests for the curl snippet with different command formatters, but there's nothing REST Assured-specific about it so I don't think they belong here.